lib/std/mem.zig

```diff
@@ -1936,25 +1937,31 @@ pub fn nativeToBig(comptime T: type, x: T) T {
     };
 }
 
+fn CopyPtrAttrs(comptime source: type, size: builtin.TypeInfo.Pointer.Size, child: type) type {
```

Comment:
Nit: `size`, `child` should also be comptime, the fact that stage1 allows this is a bug (at least for `child`)